data/src/main/scala/io/prediction/data/storage/Models.scala

```diff
@@ -78,3 +78,6 @@ class ModelSerializer extends CustomSerializer[Model](
         Nil)
   }
 ))
+
+// Use where models are saved outside the usual methods in pio
+case class NullModel()
```

Comment:
ready for review

This is to avoid using a java `null` for cases where the model is not stored in hdfs. By using this model any template can choose to store the model in a DB or as the Universal Recommender does, in Elasticsearch.